fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java

```diff
@@ -368,13 +368,15 @@ public static List<Column> toFullSchemasForHiveTable(Table table) {
         List<Column> fullSchema = Lists.newArrayList();
         for (FieldSchema fieldSchema : fieldSchemas) {
             Type type;
+            String comment = "";
             try {
+                comment = fieldSchema.getComment();
                 type = ColumnTypeConverter.fromHiveType(fieldSchema.getType());
             } catch (InternalError | Exception e) {
                 LOG.error("Failed to convert hive type {} on {}", fieldSchema.getType(), table.getTableName(), e);
                 type = Type.UNKNOWN_TYPE;
             }
-            Column column = new Column(fieldSchema.getName(), type, true);
+            Column column = new Column(fieldSchema.getName(), type, true, comment);
```

Comment:
I think it better to use column.addComent. There are already too many column ctor